axis.go

```diff
@@ -355,12 +350,22 @@ type DefaultTicks struct{}
 
 var _ Ticker = DefaultTicks{}
 
+var labels []float64
+
+var new_labels []float64
```

Comment:
why do you make this a global variable ?

AFAICT, it's only accessed inside `DefaultTicks.Ticks(min,max)`.
just declare it there?

Reply:
Changed that. Thanks!